packages/storybook/stories/CallComposite/Quickstart1ToNDocs.stories.mdx

```diff
@@ -0,0 +1,176 @@
+import { Meta, Source } from '@storybook/addon-docs';
+import Quickstart1TonSnippet from '!!raw-loader!./snippets/Quickstart1ToN.snippet.tsx';
+import { SingleLineBetaBanner } from '../BetaBanners/SingleLineBetaBanner';
+import { DetailedBetaBanner } from '../BetaBanners/DetailedBetaBanner';
+import { StartPSTNCall } from './StartPSTNCall.stories';
+import { CallComposite } from '@azure/communication-react';
+import { defaultCallCompositeHiddenControls, controlsToAdd } from '../controlsUtils';
+
+<Meta
+  id="quickstarts-1ton"
+  title="Composites/CallComposite/1:N Call/1:N Get Started"
+  parameters={{ previewTabs: { canvas: { disable: true, hidden: true } } }}
+/>
+
+# Getting Started With 1:N
+
+<DetailedBetaBanner />
+
+The UI Library CallWithChat Composite and Calling Composite supports 1:N by enabling developers to perform outbound calls through
+Azure Communication Services call service. Currently, Inbound calls (receiving calls from outbound ACS calls) is not supported by
```

Comment:
I would reword this saying that inbound scenarios aren't supported by our composite level offerings or something since we know we can build a component app that can receive inbound calls.